sql/telemetry/feature_usage/query.sql

```diff
@@ -0,0 +1,83 @@
+SELECT
+  events_t.client_id AS clients_id,
+  events_t.submission_date AS submission_date,
+  COALESCE(protection_report, FALSE) AS protection_report,
+  COALESCE(pinned_tab, FALSE) AS pinned_tab,
+  COALESCE(default_browser, FALSE) AS default_browser,
+  COALESCE(many_uri, FALSE) AS many_uri,
+  COALESCE(pip, FALSE) AS pip,
+  COALESCE(filled_password, FALSE) AS filled_password,
+  COALESCE(remember_password, FALSE) AS remember_password,
+  COALESCE(sync_configured, FALSE) AS sync_configured
+FROM
+  (
+    SELECT
+      client_id,
+      submission_date,
+      LOGICAL_OR(event_method = 'show' AND event_object = 'protection_report') AS protection_report,
+      LOGICAL_OR(event_category = 'pictureinpicture' AND event_method = 'create') AS pip,
+    FROM
+      `moz-fx-data-shared-prod.telemetry.events`
+    WHERE
+      submission_date = @submission_date
```

Comment:
What about sampling? Is that an option here?

Reply:
Not for my current application.